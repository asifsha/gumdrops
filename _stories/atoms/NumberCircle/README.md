The `<NumberCircle>` component is used to indicate numeric rankings for associated content.
