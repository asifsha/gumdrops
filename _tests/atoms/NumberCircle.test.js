/* globals mount */
import React from 'react';
import NumberCircle from '../../components/atoms/NumberCircle';

const defaultProps = {
    className: 'modal-class',
    context: 'success',
    size: 'lg',
    text: '100',
    style: { width: '100px' }
};

describe('Expect <NumberCircle>', () => {
    it('to render', () => {
        const wrapper = mount(<NumberCircle {...defaultProps} />);
        expect(wrapper).toMatchSnapshot();
    });
});
