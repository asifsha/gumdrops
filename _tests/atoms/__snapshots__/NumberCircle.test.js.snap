// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect <NumberCircle> to render 1`] = `
<NumberCircle
  className="modal-class"
  context="success"
  size="lg"
  style={
    Object {
      "width": "100px",
    }
  }
  text="100"
>
  <span
    className="gds-number-circle modal-class gds-number-circle--success gds-number-circle--lg"
    style={
      Object {
        "width": "100px",
      }
    }
  >
    100
  </span>
</NumberCircle>
`;
